pinot-spi/src/main/java/org/apache/pinot/spi/env/PinotConfiguration.java

```diff
@@ -357,6 +358,9 @@ public long getProperty(String name, long defaultValue) {
    */
   public String getProperty(String name, String defaultValue) {
     Object rawProperty = getRawProperty(name, defaultValue);
```

Comment:
For this couldn't we check if the property key exists and if not use defaultValue, otherwise just have the same logic as getProperty(String name, Configuration configuration)? That would handle the multi-value case I think

Reply:
yeah, why not. lemme update